packages/orbit-components/src/Card/components/Header.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import cx from "clsx";
+
+import type * as Common from "../../common/types";
+import type { As } from "../../Heading/types";
+import ChevronDown from "../../icons/ChevronDown";
+import Stack from "../../Stack";
+import Heading from "../../Heading";
+import Text from "../../Text";
+import Close from "../../icons/Close";
+import ButtonLink from "../../ButtonLink";
+
+type Props = {
```

Comment:
I do not mind having it as a type, but we get used to defining those types with interfaces. Here is no intersection, so there is no difference in what to use, it just looks different and not like the majority in our project. So we either do this way or another. Maybe you could justify why you find that it's better, but for me, it just looks like you do it in the way you get used to :)

Reply:
Orbit is the first project I have seen using `interface` instead of `type` 😅 `interface` provides no benefit and is more verbose. since there's no ESlint rule, I assumed both syntaxes are correct so I went with the more "terse" and more common one

Reply:
I hope you do not mind if I will enforce it with [consistent-type definitions](https://typescript-eslint.io/rules/consistent-type-definitions/) in a separate request, wdyt? :) In order to avoid such kind of cases 🙂 

I feel like we should use interfaces for object kind of type definitions, it has also the advantage of throwing an error in case of a conflict, while type alias will just merge it. And use the type aliases for everything else.

Reply:
aight let's do it in a separate request